opendemic/scheduler/confirmed_cases.py

```diff
@@ -0,0 +1,115 @@
+from config.config import CONFIG, logger
+import os
+import pandas as pd
+import uuid
+import numpy as np
+import tarfile
+from tarfile import TarFile, ExFileObject
+import requests
+import sqlalchemy
+from sqlalchemy.engine.base import Engine
+from io import BytesIO
+from typing import List
+
+_CONFIRMED_CASE_DATA_URL: str = "https://github.com/beoutbreakprepared/nCoV2019/blob/master/latest_data/latestdata.tar.gz?raw=true"
+
+
+def generate_db_uri():
```

Comment:
Would it be possible to have this function in one place in the single app and keep referencing to it?

Reply:
Good point. I'll take care of that.